src/ConfigProvider.php

```diff
@@ -4,38 +4,31 @@
 
 namespace Answear\GlsBundle;
 
-use Answear\GlsBundle\Enum\CountryCode;
+use Answear\GlsBundle\Enum\CountryCodeEnum;
 
 class ConfigProvider
 {
-    private const URL_MAP = [
-        CountryCode::HUNGARY => 'https://online.gls-hungary.com/',
-        CountryCode::SLOVAKIA => 'https://online.gls-slovakia.com/',
-        CountryCode::CZECH => 'https://online.gls-czech.com/',
-        CountryCode::ROMANIA => 'https://online.gls-romania.com/',
-        CountryCode::SLOVENIA => 'https://online.gls-slovenia.com/',
-        CountryCode::CROATIA => 'https://online.gls-croatia.com/',
-    ];
-
-    private CountryCode $countryCode;
+    private CountryCodeEnum $countryCode;
 
     public function __construct(string $countryCode)
     {
-        $this->countryCode = CountryCode::byValue($countryCode);
+        $this->countryCode = CountryCodeEnum::from($countryCode);
     }
 
-    public function getCountryCode(): CountryCode
+    public function getCountryCode(): CountryCodeEnum
     {
         return $this->countryCode;
     }
 
     public function getUrl(): string
     {
-        $url = self::URL_MAP[$this->getCountryCode()->getValue()] ?? null;
-        if (null === $url) {
-            throw new \InvalidArgumentException('No url for provided country code.');
-        }
-
-        return $url;
+        return match ($this->getCountryCode()) {
+            CountryCodeEnum::Romania,
+            CountryCodeEnum::Hungary => 'https://map.gls-hungary.com/',
+            CountryCodeEnum::Slovakia => 'https://map.gls-slovakia.com/',
+            CountryCodeEnum::Czech => 'https://map.gls-czech.com/',
+            CountryCodeEnum::Slovenia => 'https://map.gls-slovenia.com/',
+            CountryCodeEnum::Croatia => 'https://map.gls-croatia.com/',
+        };
```

Comment:
Why this is better?
I preferred const for this map

Reply:
I fixed it